lib/sagemaker-model/container-images.ts

```diff
@@ -24,8 +24,10 @@ export class ContainerImages {
     "huggingface-pytorch-tgi-inference:2.0.1-tgi1.0.3-gpu-py39-cu118-ubuntu20.04";
   static readonly HF_PYTORCH_LLM_TGI_INFERENCE_1_1_0 =
     "huggingface-pytorch-tgi-inference:2.0.1-tgi1.1.0-gpu-py39-cu118-ubuntu20.04";
+  static readonly HF_PYTORCH_LLM_TGI_INFERENCE_1_3_3 =
+    "huggingface-pytorch-tgi-inference:2.1.1-tgi1.3.3-gpu-py310-cu121-ubuntu20.04";
   static readonly HF_PYTORCH_LLM_TGI_INFERENCE_LATEST =
-    ContainerImages.HF_PYTORCH_LLM_TGI_INFERENCE_1_1_0;
+    ContainerImages.HF_PYTORCH_LLM_TGI_INFERENCE_1_3_3;
```

Comment:
Let's not update/use LATEST as it might impact other models running on previous TGI version unexpectedly. To be fair we should remove LATEST from this but it's out of scope of this PR. 

So please just change mi(s|x)tral models to point to the explicit version instead of LATEST.

Reply:
have done this and squashed commits